### PR TITLE
Insert information

### PR DESCRIPTION
**Pull request by sonianb**
User can add new ice cream and the info will be displayed on a new page.

### CODE REVIEW

**Review by mariaalouisaa — COMMENTED** [review]
👍

File: routes/home.js [hunk 1]
```diff
@@ -56,24 +56,32 @@ function get(request, response) {
...
 
-  db.query(insert_user, values).then(() => {
-    //Seems like things are stopping here....
-    console.log("query check");
-    response.redirect("/show-posts"); 
+  
+  db.query(`INSERT INTO users(username, age, fandom) VALUES($1, $2, $3) RETURNING id`, [request.body.username, request.body.age, request.body.fandom]).then((newUser) => {
+    // console.log(newUser) will return everything that has changed in the database, returns an array of all the elems
+    const id = newUser.rows[0].id; 
+    return db.query(`INSERT INTO ice_cream_posts(user_id, base_flavour, topping, comment) VALUES($1, $2, $3, $4)`, [id, request.body.base_flavour, request.body.topping, request.body.comment]).then(() => {
```
Comment (PJSalter) [inline routes/home.js:79]: AMAZING 🤩 Thank you for getting this fixed ✨

**Comment by lascellesabercrombie** [comment]
Looks good!

File: routes/home.js [hunk 2]
```diff
@@ -56,24 +56,32 @@ function get(request, response) {
...
+      response.redirect("/show-posts")
+    }).catch((err) => {
```
Comment (PJSalter) [inline routes/home.js:81]: great use of catch, neatly implemented ✨

**Review by PJSalter — APPROVED** [review]

**Pull request merged**